server/constants.py

```diff
@@ -16,7 +16,11 @@
 
 # Game constants
 # DEFAULT_GAME = 'quoridor'
+# GAME_HOST_PORT = "QUORIDOR_HOST_PORT"
+# GAME_PORT = "localhost:50051"
```

Comment:
Are these comments necessary?

Reply:
Is only to change the game more quickly. If you considered it unnecessary I can delete it

Reply:
Sometimes comments are never removed, that's why I pointed it out.

Reply:
it is understood. to avoid bad practices delete the comments